update-images/action.yml

```
@@ -0,0 +1,45 @@
+name: Update Images
+description: Bulk update images for multiple services using duploctl
+author: 'duplocloud'
+branding:
+  icon: 'layers'
+  color: 'blue'
+inputs:
+  services:
+    description: |
+      JSON array of service-image pairs.
+      Format: [{"service": "service1", "image": "image1:tag"}]
+    required: true
+  wait:
+    description: Wait for all deployments to complete
+    required: false
+    default: "false"
+
+runs:
+  using: composite
+  steps:
+
+  - name: Bulk Update Images
+    id: bulk-update
+    shell: bash
+    env:
+      SERVICES: ${{ inputs.services }}
+      WAIT: ${{ inputs.wait }}
+    run: |
+      # Build the duploctl command
```

Suggesting some fail fast checks on dependencies:
```suggestion
      set -euo pipefail
      
      # Check dependencies
      command -v jq >/dev/null || { echo "jq is required"; exit 1; }
      command -v duploctl >/dev/null || { echo "duploctl is required"; exit 1; }
      # Build the duploctl command
```